test/uk/gov/hmrc/entrydeclarationdecision/connectors/StoreConnectorSpec.scala

```diff
@@ -241,4 +247,22 @@ class StoreConnectorSpec
       }
     }
   }
+
+  "StoreConnector.setShortTtl" when {
+    val url: String = s"/import-control/housekeeping/submissionid/$submissionId"
+    "successful" must {
+      "return Unit" in new Test {
+        stubPutRequest(url, NO_CONTENT)
+        val result: Unit = await(connector.setShortTtl(submissionId))
+        result shouldBe ():Unit
+      }
+    }
+    "unsuccessful" must {
+      "return Unit" in new Test {
+        stubPutRequest(url, NOT_FOUND)
+        val result: Unit = await(connector.setShortTtl(submissionId))
+        result shouldBe ():Unit
```

Comment:
Both tests are essentially testing that the request is made and that the connector does not throw an exception or return a failed future. I.e. the result cannot be anything other than the Unit singleton value.

Not sure but it feels like it should either just await(...) and be done. In which case in the failure scenario description is maybe that the connector explicitly ignores the failure.

Or else maybe have a boolean return value. Hmm...